SendStruct/client.c

```diff
@@ -0,0 +1,72 @@
+#include <arpa/inet.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#define PORT 8080
+
+struct Test {
+  int Id;
```

Comment:
Should be id. Variable should be naming using lower case first.

Reply:
I renamed it